src/xdocs/writingchecks.xml

```diff
@@ -422,7 +422,7 @@ public class MethodLimitCheck extends AbstractCheck
     <section name="Comment Tokens in AST">
 
       <p>
-        Before <a href="releasenotes.html#Release_6.0">Checkstyle 6.0</a>, there was no comments
+        Before <a href="releasenotes_old.html#Release_6.0">Checkstyle 6.0</a>, there was no comments
```

Comment:
Should we apply same naming concept for this file ?

Reply:
Done.